### DISCUSSION TITLE
LOGB Metadata in GPMF

### DISCUSSION THREAD

**Discussion by RikshaDriver** [gopro/labs#545] [Q&A] [answered]
I've had a look through the Metadata Extractor for some files shot on H11 Mini with custom LogB parameters. I could see some data under DEVC and some under STRM:

  STRM 
    EXPS 1
    NR01 1
    BITR 200
    24HZ 1
    EXPN 10000
    EXPX 48
    USRD 1

But I couldn't find LOGB values. Is this information not captured at the moment?

**Comment by dnewman-gpsw** [comment]
If you set it, it should be stored. Only passwords are hidden. If commands aren't stored, try !RESET and store again.

**Comment by RikshaDriver** [comment]
OK, looks like a mistake on my end... Used the wrong saved preset for some clips.

After some investigation though, I note that gpmf-parser is unable to show LOGB values. I'm guessing there may need to be some code updates to support some of the newer FourCC parameters?

Also another interesting note is that the Metadata Extractor is showing incorrect offset values... for example offset of -128 is showing as:
LOGB 200, 4294967168

**Reply by dnewman-gpsw** [reply]
I can't think of any reason gpmf-parser wouldn't support LOGB.

The LOGB 200, 4294967168  is signed number display error on the webpage, now fixed.  Try it again.

**Reply by RikshaDriver** [reply]
Thanks for that. It's working well.

Regarding gpmf-parser... here's an extract:

gpmf-parser GX010003.MP4 -fLOGB
VIDEO FRAMERATE:
  24.000 with 398 frames
PAYLOAD TIME:
  0.000 to 1.000 seconds
SCALED DATA:
COMPUTED SAMPLERATES:
  ACCL sampling rate = 200.795242Hz (time 0.001714 to 16.625614)",
  GYRO sampling rate = 200.795242Hz (time 0.001714 to 16.625614)",
  SHUT sampling rate = 23.993229Hz (time 0.000000 to 16.588013)",
  WBAL sampling rate = 8.000000Hz (time 0.000000 to 16.625000)",
  WRGB sampling rate = 8.000000Hz (time 0.000000 to 16.625000)",
  ISOE sampling rate = 23.993229Hz (time 0.000000 to 16.588013)",
  YAVG sampling rate = 8.000000Hz (time 0.000000 to 16.625000)",
  UNIF sampling rate = 8.000000Hz (time 0.000000 to 16.625000)",
  SCEN sampling rate = 8.000000Hz (time 0.000000 to 16.625000)",
  HUES sampling rate = 8.000000Hz (time 0.000000 to 16.625000)",
  FACE sampling rate = 12.000000Hz (time 0.000000 to 16.583333)",
  GPS5 sampling rate = 10.000605Hz (time 0.045271 to 16.644267)",
  GPS9 sampling rate = 10.000605Hz (time 0.045271 to 16.644267)",
  CORI sampling rate = 23.993229Hz (time 0.000000 to 16.588013)",
  IORI sampling rate = 23.993229Hz (time 0.000000 to 16.588013)",
  GRAV sampling rate = 23.993229Hz (time 0.000000 to 16.588013)",
  WNDM sampling rate = 10.000000Hz (time 0.099532 to 16.599532)",
  MWET sampling rate = 10.000000Hz (time 0.099532 to 16.599532)",
  AALP sampling rate = 10.000000Hz (time 0.099532 to 16.599532)",
  MSKP sampling rate = 23.993229Hz (time 0.000000 to 16.588013)",
  LSKP sampling rate = 23.993229Hz (time 0.000000 to 16.588013)",


This particular clip has LOGB 200,-128



On a side note, just following up if you've had a chance to respond to my emails :)

**Reply by dnewman-gpsw** [reply, marked as answer]
That is not a gpmf-parser failure, the demo code is compiled to either get the streaming time-change data, or the header GPMF data.  LOGB is not time-changing, so it is in the header.
